e2e/playwright/point-click.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { test } from './authenticatedAppFixture'
+
+// test file is for testing point an click code gen functionality that's not sketch mode related
+
+test('verify extruding circle works', async ({ app }) => {
+  const file = await app.getInputFile('test-circle-extrude.kcl')
+  await app.initialise(file)
+  const [clickCircle, moveToCircle] = app.makeMouseHelpers(582, 217)
```

Comment:
tuple pattern is nice since it makes renaming them easy, which I think since we need mouse moves to arbitrary coords, should help us keep the rest of the test readable.